Assets/BossRoom/Scripts/Client/UI/PopupPanel.cs

```diff
@@ -13,72 +13,38 @@ public class PopupPanel : MonoBehaviour
         TextMeshProUGUI m_TitleText;
         [SerializeField]
         TextMeshProUGUI m_MainText;
+        [SerializeField]
+        CanvasGroup m_CanvasGroup;
 
-        bool m_IsPopupShown;
-
-        static PopupPanel s_Instance;
+        public bool IsDisplaying => m_CanvasGroup.alpha > 0;
```

Comment:
not a fan of implicit meaning like this. (canvasGroup alpha == "IsDisplaying --> what happens when we change our structure and canvasGroup isn't representative of the whole popup anymore?
Having an explicit IsDisplaying = false in Hide() would make this clearer